pkgs/top-level/all-packages.nix

```diff
@@ -15879,8 +15879,7 @@ with pkgs;
   openjdk = jdk;
   openjdk_headless = jdk_headless;
 
-  graalvmCEPackages = callPackage ../development/compilers/graalvm/community-edition { };
-  graalvm-ce = graalvmCEPackages.graalvm-ce;
+  graalvmPackages = recurseIntoAttrs (callPackage ../development/compilers/graalvm { });
```

Comment:
I don't like the usage of `recurseIntoAttrs` here, we probably need to introduce a scope inside `graalvmPackages` to ensure that all packages are build with the same scope.

Reply:
I'm not familiar with scopes and not sure what this means. I tried just removing `recurseIntoAttrs` and things seem to work all the same, so it seems it can be safely dropped?

Is the fact that `pkgs/development/compilers/graalvm/graalvm-oracle/default.nix` uses `graalvmPackages.buildGraalvm` (from the parent "scope"?) problematic?